### PR TITLE
[No Ticket] Add Grunt task to quickly lint and typecheck ts files

### PR DESCRIPTION
**Pull request by zetagame**
### This PR will...
Add a grunt task to check TS files.

#### Addresses Issue(s):

JW8-###

### Checklist
- [ ] Jenkins builds and unit tests are passing
- [ ] I have reviewed the automated results

### CODE REVIEW

File: Gruntfile.js [hunk 1]
```diff
@@ -264,6 +264,11 @@ module.exports = function(grunt) {
         'karma:local'
     ]);
 
+    grunt.registerTask('ts', [
```
Comment (robwalch) [inline Gruntfile.js]: I would prefer that this be added as a script task in `project.json`.

- `yarn ts` or `npm run ts` to run
- Less reliance on grunt
- We can still run script tasks in grunt by calling to `yarn` ci

Comment (jnatalzia) [reply Gruntfile.js]: i would be down to add to `yarn type-check`

File: package.json [hunk 2]
```diff
@@ -84,7 +84,8 @@
     "squash": "node parse-squashed-files.js",
     "test": "karma start",
     "karma:hook": "karma start --browsers=ChromeHeadless --reporters=mocha",
-    "type-check": "tsc"
+    "type-check": "tsc",
+    "ts-check": "tsc && eslint './src/js' --ext .ts --ext .js"
```
Comment (jnatalzia) [inline package.json]: `--ext .ts` only?

Comment (zetagame) [reply package.json]: good call

**Pull request merged**